frontend/src/pages/Login.jsx

```diff
@@ -53,6 +54,7 @@ const LoginPage = () => {
             </Button>
           </Form.Item>
         </Form>
+        <text>don't have an account yet ? <text onClick={() => navigate("Register")}>Sign Up here</text></text>
```

Comment:
The use of `<text>` tag is not valid in React. Instead, you should use `<span>` or `<div>` for text within JSX.
```suggestion
         <div>Don't have an account yet? <span onClick={() => navigate("Register")}>Sign Up here</span></div>
```